docs/book/v5/introduction/packages.md

```diff
@@ -1,7 +1,7 @@
 # Packages
 
 * `dotkernel/dot-dependency-injection` - Dependency injection component using class attributes.
-* `dotkernel/dot-authorization` - Authorization base package defining interfaces for authorization services to be used with DotKernel applications
+* `dotkernel/dot-authorization` - Authorization base package defining interfaces for authorization services to be used with Dotkernel applications
 * `dotkernel/dot-controller` - Provides base classes for action based controllers similar to Laminas controller component
 * `dotkernel/dot-data-fixtures` - Provides a CLI interface for listing & executing doctrine data fixtures
 * `dotkernel/dot-debugbar` - Wrapper on top of maximebf/php-debugbar
```

Comment:
please check against the current composer.json file if debugbar is there or not 
anyway , please re-check all libraries to have a match

Reply:
removed dotkernel/dot-debugbar and sorted alphabetically